flink-clients/src/main/java/org/apache/flink/client/python/PythonUtil.java

```diff
@@ -0,0 +1,200 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.client.python;
+
+import org.apache.flink.core.fs.FileSystem;
+import org.apache.flink.core.fs.Path;
+import org.apache.flink.util.FileUtils;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.FileSystems;
+import java.nio.file.Files;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+
+/**
+ * The util class help to prepare Python env and run the python process.
+ */
+public final class PythonUtil {
+	private static final Logger LOG = LoggerFactory.getLogger(PythonUtil.class);
+
+	private static final String FLINK_OPT_DIR = System.getenv("FLINK_OPT_DIR");
+
+	private static final String FLINK_OPT_DIR_PYTHON = FLINK_OPT_DIR + File.separator + "python";
+
+	private static final String PYFLINK_LIB_ZIP_FILENAME = "pyflink.zip";
+
+	private static final String PYFLINK_PY4J_FILENAME = "py4j-0.10.8.1-src.zip";
```

Comment:
It's better to auto find the `xxx.zip` and `py4j.xxx-src.zip` in dist folder `opt/python`, What do you think?

Reply:
Yes, it looks better.I have added code about finding the python job needed libs in opt directory.Thank you for advice.